src/utils/common.ts

```diff
@@ -323,106 +322,6 @@ async function getLineaGasFees(chainId: number, transport: Transport | undefined
   };
 }
 
-export type UpdateDepositDetailsMessageType = {
-  UpdateDepositDetails: [
-    {
-      name: "depositId";
-      type: "uint32";
-    },
-    { name: "originChainId"; type: "uint256" },
-    { name: "updatedRelayerFeePct"; type: "int64" },
-    { name: "updatedRecipient"; type: "address" },
-    { name: "updatedMessage"; type: "bytes" },
-  ];
-};
-
-export type UpdateV3DepositDetailsMessageType = {
-  UpdateDepositDetails: [
-    { name: "depositId"; type: "uint32" },
-    { name: "originChainId"; type: "uint256" },
-    { name: "updatedOutputAmount"; type: "uint256" },
-    { name: "updatedRecipient"; type: "address" },
-    { name: "updatedMessage"; type: "bytes" },
-  ];
-};
-
-/**
- * Utility function to get EIP-712 compliant typed data that can be signed with the JSON-RPC method
- * `eth_signedTypedDataV4` in MetaMask (https://docs.metamask.io/guide/signing-data.html). The resulting signature
- * can then be used to call the method `speedUpDeposit` of a `SpokePool.sol` contract.
- * @param depositId The deposit ID to speed up.
- * @param originChainId The chain ID of the origin chain.
- * @param updatedRelayerFeePct The new relayer fee percentage.
- * @param updatedRecipient The new recipient address.
- * @param updatedMessage The new message that should be provided to the recipient.
- * @return EIP-712 compliant typed data.
- */
-export function getUpdateDepositTypedData(
-  depositId: number,
-  originChainId: number,
-  updatedRelayerFeePct: BigNumber,
-  updatedRecipient: string,
-  updatedMessage: string
-): TypedMessage<UpdateDepositDetailsMessageType> {
-  return {
-    types: {
-      UpdateDepositDetails: [
-        { name: "depositId", type: "uint32" },
-        { name: "originChainId", type: "uint256" },
-        { name: "updatedRelayerFeePct", type: "int64" },
-        { name: "updatedRecipient", type: "address" },
-        { name: "updatedMessage", type: "bytes" },
-      ],
-    },
-    primaryType: "UpdateDepositDetails",
-    domain: {
-      name: "ACROSS-V2",
-      version: "1.0.0",
-      chainId: originChainId,
-    },
-    message: {
-      depositId,
-      originChainId,
-      updatedRelayerFeePct,
-      updatedRecipient,
-      updatedMessage,
-    },
-  };
-}
-
-export function getUpdateV3DepositTypedData(
```

Comment:
We use this [here](https://github.com/across-protocol/frontend/blob/7a57b128a9730b8b3009ecc6317523bc3c99988a/src/views/Transactions/hooks/useSpeedUp.tsx#L71) in the FE, so do we want to remove this now?

Reply:
The FE has its own definition of this we use: https://github.com/across-protocol/frontend/blob/7a57b128a9730b8b3009ecc6317523bc3c99988a/src/utils/sdk.ts#L17

Reply:
Ah I see.